sql/src/test/java/org/apache/druid/sql/calcite/CalciteQueryTest.java

```diff
@@ -2265,6 +2265,82 @@ public void testNullEmptyStringEquality() throws Exception
     );
   }
 
+  @Test
+  public void testNullLongFilter() throws Exception
+  {
+    // bug in numeric vector matches prevents this test from passing
```

Comment:
Please raise an issue for the bug and put the github URL here. (Same with the others)

Reply:
linked to issue with bug, also will be fixed by #9063